CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.0] - 2020-01-TBD
+
+### Added
+
+The release switches us over to using gRPC-Web instead of the node.js version of gRPC. The goal of this change is to enabled the library to be used in browser as well as on Node.js without having two separate codebases or build strategies.
```

Comment:
"us over to using" -> "this library to using"

Generally prefer to refer to things objectively rather than the collective 'we'. There's probably some named grammar rule for this.